src/features/auth/ui/SigninForm.tsx

```
@@ -16,134 +14,77 @@ const SigninForm = () => {
-                height={10}
-                priority
-                className="mb-[2px]"
-              />
-              <span className="text-[10px] font-[700] text-[#EC3030]">{errorMessage}</span>
-            </div>
-          )}{' '}
-          {/* 6->4, 15->10, 3->2, 15->10 */}
+    <div className="space-y-5">
+      <div>
+        <label className="block text-sm font-medium text-gray-300 mb-2">이메일</label>
+        <div className="relative">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)